### ISSUE TITLE
Add images

### ISSUE THREAD

**Issue by tirthbha** [ubsuny/23-Homework1G2#88]
@Pranjal-Srivastava-2023 @s4il3sh 
Can we add image files to the main branch that we have added to the documentation?

**Comment by Pranjal-Srivastava-2023** [comment]
@tirthbha No! do not add anything to main branch. Always create a separate branch and send merge request. I see you added a png file directly to the main repository.
